cluster/manifests/efs-provisioner/cm-efs-provisioner.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: efs-provisioner
+  namespace: kube-system
+data:
+  file.system.id: "{{ .ConfigItems.efsid }}"
```

Comment:
Maybe all these resources should only be deployed to clusters which has this item defined.

Reply:
I updated it to only deploy is `efsid` is set. This way we can enable for selected clusters.